src/components/map/LayersService.js

```diff
@@ -334,15 +334,14 @@ goog.require('ga_urlutils_service');
                     ' swisstopo',
                 subLayersIds: [
                   'OpenMapTiles',
-                  'ch.swisstopo.swissalti3d-reliefschattierung',
-                  'ch.swisstopo.vektorkarte.vt',
+                  'ch.swisstopo.vektorkarte_aggregated.vt',
                   'ch.bav.haltestellen-oev.vt',
                   'ch.swisstopo.amtliches-strassenverzeichnis_validiert',
                   'ch.swisstopo.swissnames3d.vt'
```

Comment:
What it aggregated ?  haltestelle, swissnames and strassen?  in that case you can remove the corresponding layers from the subLayersIds list.